{"name": "Создание заказа с неверным хешем ингредиента", "status": "passed", "start": 1706479165179, "stop": 1706479165423, "uuid": "47b84e0b-b745-4129-b400-e45bd1484f5a", "historyId": "e9f909289041bf7772ca1eca0e2bcee9", "testCaseId": "e9f909289041bf7772ca1eca0e2bcee9", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_invalid_hash_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "DESKTOP-29U91I9"}, {"name": "thread", "value": "24328-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}